### PR TITLE
BTMKPL - Update postal code providers

### PR DESCRIPTION
**Pull request by mjrossi**
- Add `de_AT` postal_code provider with format: ####
- Add `de_CH` postal_code provider with format: ####
- Update `el_GR` postal_code provider: remove TK prefixes as a valid
  postal code.

### CODE REVIEW

File: faker/tests/el_GR/__init__.py [hunk 1]
```diff
@@ -0,0 +1,19 @@
+from __future__ import unicode_literals
+
+import unittest
+import re
+
+from faker import Factory
+from faker.utils import text
+from .. import string_types
+
+
```
Comment (cmonty) [inline faker/tests/el_GR/__init__.py]: Extra newline.

**Comment by cmonty** [comment]
:+1:

File: faker/tests/el_GR/__init__.py [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+class el_GR_FactoryTestCase(unittest.TestCase):
+    def setUp(self):
+      self.factory = Factory.create('el_GR')
+
+    def test_postcode(self):
+        postcode  = self.factory.postcode()
```
Comment (ryanjpodonnell) [inline faker/tests/el_GR/__init__.py]: indentation in this method is not conistent with the method above

**Comment by ryanjpodonnell** [comment]
💍 🔔 💍 🔔 💍 🔔

**Pull request merged**